options/posix/include/sys/fcntl.h

```diff
@@ -0,0 +1 @@
+#include <fcntl.h>
```

Comment:
Please add the include guards, even if they aren't especially needed.

Reply:
IMO include guards on passthrough headers are needless complexity and also not something we do so far, see for instance `sys/syslog.h`.